docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
+Extension to use for your build with OLED Display
+
+If you need help soldering, there's a small note on the end to help.
+
+# Preparation
+
+### Libraries
+First of all you need to download few Libraries that will make possible for your OLED to work. You can download them from Adafruit CircuitPython Libraries page. 
```

Comment:
```suggestion
First of all you need to download a few libraries that will make it possible for your OLED to work.
You can get them with the [Adafruit CircuitPython Libraries bundle](https://circuitpython.org/libraries).
```
* Adhere to standard english grammar. Let us know if you would like us to assist with that.
* Preferably use one line per sentence.
* This is where the link 2 lines down should go.

Reply:
Let me know if there are still language errors somewhere. English is not my native language and sometimes I have problems.